Library/TeamTalk_DLL/TeamTalk.h

```diff
@@ -6915,6 +6915,40 @@ extern "C" {
                                                           IN StreamType nStreamType,
                                                           IN INT32 nDelayMSec);
 
+     /**
+     * @brief Set the configuration for de-jitter measures for a user.
+     *
+     * TeamTalk can add a fixed delay at the start of the playout of a user stream.
+     * This delay acts as a buffer to smooth out jittering (non-constant delays)
+     * in the reception of network packets.
+     * The fixed delay is applied at the start of every new stream, such as a new PTT session.
+     * The default fixed delay is zero.
+     *
+     * TeamTalk can also apply adaptive jitter buffering where the actual jitter is measured
+     * and the delay at the start of a stream is adapted based on those measurements.
+     * The adaptive delay will not go below the fixed delay.
+     * The parameter nMaxAdaptiveDelayMSec maximizes the total adaptive delay.
+     * By default, the adaptive mechanism is OFF
+     *
+     * By default, all jitter control is OFF
+     * 
+     * The result of jitter buffering is that playout frames will get buffered in the playout buffer.
+     * Make sure to also size the playout buffer for the expected jitter via #TT_SetUserAudioStreamBufferSize
+     *
+     * @param lpTTInstance Pointer to client instance created by #TT_InitTeamTalk.
+     * @param nUserID The user ID of the user to apply the configuration to.
+     * @param nStreamType The type of stream to change, currently only
+     * #STREAMTYPE_VOICE is supported. Other types are a no-op.
+     * @param nFixedDelayMSec The fixed delay in milliseconds. Default = 0.
+     * @param bMute Turns adaptive jitter buffering ON/OFF. Default is OFF.
+     * @param nMaxAdaptiveDelayMSec A hard maximum on the adaptive delay.*/
+     TEAMTALKDLL_API TTBOOL TT_SetUserJitterControl(IN TTInstance* lpTTInstance,
+                                                    IN INT32 nUserID,
+                                                    IN StreamType nStreamType,
+                                                    IN INT32 nFixedDelayMSec,
```

Comment:
Should we use a struct here instead for the last three parameters? Then it's easier to extend later with more features.

Reply:
Changed this in an additional commit. 
I have no idea why the build on Github fails. The test that fails runs perfectly on this jitter_buffer branch as well as the master branch on Ubuntu18.04 and Windows10